webapp/app/_components/header/header.tsx

```diff
@@ -3,34 +3,45 @@ import Button from "@/components/button/button";
 
 const Header = () => {
   return (
-    <header className="flex justify-between items-center px-32 py-4 bg-[#211E27]">
+    <header className="flex justify-between items-center px-24 py-4 bg-[#24202A] shadow-md">
       <Image
         src="/resume_matcher_logo.png"
         alt="Resume Matcher Logo"
-        width={250}
-        height={90}
+        width={100}
+        height={60}
       />
       <nav className="grow">
-        <ul className="flex justify-center gap-6 text-[#DCCDCD]" role="list">
-          <li>Contributions</li>
-          <li>Docs</li>
-          <li>Main Site</li>
+        <ul
+          className="flex justify-center gap-6 text-[#e2e8f0] text-sm "
+          role="list"
+        >
+          <li className="hover:text-[#94a3b8]">
+            <a href="https://github.com/srbhr/Resume-Matcher/graphs/contributors">
+              Contributions
+            </a>
+          </li>
+          <li className="hover:text-[#94a3b8]">
+            <a href="https://github.com/srbhr/Resume-Matcher#readme">Docs</a>
+          </li>
+          <li className="hover:text-[#94a3b8]">
+            <a href="https://www.resumematcher.fyi/">Main Site</a>
+          </li>
```

Comment:
I further update the links to open in new tabs, rather than opening within the same tab, which results in navigating the user away from the tool itself. The second reason is that these are external links, and so would normally open in new tabs, rather than the existing tab.

You may use the additional link attributes `target="_blank" rel="noopener noreferrer"` inside the anchor elements.

See example suggestion below:

```suggestion
          <li className="hover:text-[#94a3b8]">
            <a href="https://github.com/srbhr/Resume-Matcher/graphs/contributors" target="_blank" rel="noopener noreferrer">
              Contributions
            </a>
          </li>
          <li className="hover:text-[#94a3b8]">
            <a href="https://github.com/srbhr/Resume-Matcher#readme" target="_blank" rel="noopener noreferrer">Docs</a>
          </li>
          <li className="hover:text-[#94a3b8]">
            <a href="https://www.resumematcher.fyi/" target="_blank" rel="noopener noreferrer">Main Site</a>
          </li>
```